ddtrace/opentracer/span_context.py

```diff
@@ -1,12 +1,24 @@
+from typing import Any
+from typing import Dict
+from typing import Optional
+
 from opentracing import SpanContext as OpenTracingSpanContext
 
 from ddtrace.context import Context as DatadogContext
+from ddtrace.internal.compat import NumericType
 
 
 class SpanContext(OpenTracingSpanContext):
     """Implementation of the OpenTracing span context."""
 
-    def __init__(self, trace_id=None, span_id=None, sampling_priority=None, baggage=None, ddcontext=None):
+    def __init__(
+        self,
+        trace_id=None,  # type: Optional[int]
+        span_id=None,  # type: Optional[int]
+        sampling_priority=None,  # type: Optional[NumericType]
+        baggage=None,  # type: Optional[Dict[str, Any]]
+        ddcontext=None,  # type: Optional[DatadogContext]
+    ):
```

Comment:
```suggestion
    ):
    # type: (...) -> ?
```

Do we need one of these?